Joystream-Role-Bot/src/controls/control.ts

```diff
@@ -67,7 +67,7 @@ export const setMemberRole = async (client: Client): Promise<void> => {
     return;
   }
 
-  await api?.rpc.chain.subscribeNewHeads((header) => {
+  await api?.rpc.chain.subscribeNewHeads((header: any) => {
```

Comment:
Why were all those `any`added?

Reply:
This is required to correct Typescript warnings.